chall02/jcervill.py

```diff
@@ -0,0 +1,82 @@
+import sys
```

Comment:
You forgot the sheebang, even if the file is executable, the system won't know how to execute the script!
```suggestion
#!/usr/bin/env python3
import sys
```